src/config/index.js

```diff
@@ -13,7 +13,8 @@ export const IOTA_API = Object.freeze({
   PROVIDER_A: "http://eugene.iota.community:14265",
   PROVIDER_B: "http://eugene.iotasupport.com:14999",
   PROVIDER_C: "http://eugeneoldisoft.iotasupport.com:14265",
-  ADDRESS_LENGTH: 81
+  ADDRESS_LENGTH: 81,
+  MESSAGE_LENGTH: 2187
```

Comment:
We might want to use 2186.  iirc, when the client goes to download, there's some kind of error when converting out of trytes if the number of characters is odd, I think?

Reply:
Also, is this message length the length of the message before trytes conversion or after?

Reply:
i tested it with 2187 and it was successful. this is the length of the message after tryting